src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/config.jelly

```diff
@@ -87,6 +87,10 @@
                 <f:select/>
             </f:entry>
 
+            <f:entry title="${%Maximum_Deployment_Size}" field="maximumDeploymentSize">
+                <f:textbox/>
```

Comment:
this should use a number tag, and I think we should set a sensible default on it, at least in the UI
```suggestion
                <f:number default=10 />
```

Reply:
I would think a default of 0 would be appropriate? That would maintain existing behavior.

Reply:
This would not affect existing configurations, it would only affect new configuration.

I've certainly hit this issue before... I think setting something is better than nothing

Reply:
Oh right, because this is just the HTML so this default would only be used when adding a new template in the UI.